keyboards/lfgberg/cumpad/config.h

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2020 Lfgberg
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4C47
+#define PRODUCT_ID      0x0001
+#define DEVICE_VER      0x0001
+/*Product Info*/
+#define MANUFACTURER    Lfgberg
+#define PRODUCT         Lfgberg/Cumpad
+
+#define TAPPING_TERM 400
```

Comment:
Actually, could this be moved to a `post_config.h` file?  This way, it doesn't have to be undefined if a user wants to change it.